### ISSUE TITLE
MVP DX actions including continuous deployment

### ISSUE THREAD

**Issue by hannelita** [datopian/aircan#66]
## Acceptance
- [ ] Team adopting DX as dev/staging environment
- [ ] README instructions (simplified version)

**Comment by mbeilin** [comment]
## getting started with aircan-dev DX

We have two main repos:

1. https://gitlab.com/viderum/cloud-ckan-aircan
   Cloude image: standard CKAN portal + [ckanext-aircan](https://github.com/datopian/ckanext-aircan) installed in it.


2. https://gitlab.com/datopian/tech/dx-helm-ckan-aircan
   Helm chart managing the cloud image from above.
   
Ckan portal url:https://ckan.aircan.dev.datopian.com/

Basically, once you update the cloud image, e.g update ckanext-aircan version in [Dockerfile](https://gitlab.com/viderum/cloud-ckan-aircan/-/blob/master/Dockerfile#L157), after the pipeline finished succesfully, the Helm chart (dx-helm-ckan-aircan) should synchronize with your last changes(by pulling latest [image](https://gitlab.com/viderum/cloud-ckan-aircan/container_registry)) automatically.

**Comment by hannelita** [comment]
@mbeilin I don't access to the first URL

**Comment by hannelita** [comment]
@mbeilin Could you give a bit more detailed instructions? Like, imagine I have just jumped into the aircan project. How/where can I find instructions to update the cloud imagine in order to test a new feature I'm implementing? If you could add a few more sentences with these details that'd be awesome!

**Comment by mbeilin** [comment]
> @mbeilin I don't access to the first URL

@hannelita please try now

**Comment by hannelita** [comment]
> > @mbeilin I don't access to the first URL
> 
> @hannelita please try now

Now it works! 🎉

**Comment by mbeilin** [comment]
> @mbeilin Could you give a bit more detailed instructions? Like, imagine I have just jumped into the aircan project. How/where can I find instructions to update the cloud imagine in order to test a new feature I'm implementing? If you could add a few more sentences with these details that'd be awesome!

@hannelita but the nature of DX is all about hosting project with all its dependencies (aka ckan plugins) and 
to react properly to changes you push to this repo (https://gitlab.com/viderum/cloud-ckan-aircan). 
So you will continue working on [ckanext-aircan](https://github.com/datopian/ckanext-aircan), 
push the changes, 
add a new tag, 
update version of the extension in [Dockerfile](https://gitlab.com/viderum/cloud-ckan-aircan/-/blob/master/Dockerfile#L157),
the pipeline will start automatically,
once the pipeline finished,  the latest image will be deployed to the cluster by [dx-helm-ckan-aircan](https://gitlab.com/datopian/tech/dx-helm-ckan-aircan),
and you should be able to get the portal with your changes `ckanext-aircan` in this particular case.

From what I see it's more reliable for staging/prod envs (aka GitOps)

**Comment by mbeilin** [comment]
> > > @mbeilin I don't access to the first URL
> > 
> > 
> > @hannelita please try now
> 
> Now it works!

cloud images repos are private

**Comment by shubham-mahajan** [comment]
@hannelita @mbeilin It is throwing internal server error now.

**Comment by hannelita** [comment]
@mbeilin is there a way to see the logs?

**Comment by mbeilin** [comment]
@hannelita I'm using ArgoCD web UI for getting logs. 
http://tech.datopian.com/dx/how-to-use-the-dx-setup/#install-argo-cd-in-the-cluster

**Comment by mbeilin** [comment]
@hannelita  I've created an issue in DevOps for 500 error we are getting here:
https://gitlab.com/datopian/tech/devops/-/issues/206

**Comment by hannelita** [comment]
@shubham-mahajan Is there any update on this?

**Comment by shubham-mahajan** [comment]
@hannelita @mbeilin The Aircan instance is up and running now.

**Comment by hannelita** [comment]
@shubham-mahajan Are there any docs to access the instance?

**Comment by hannelita** [comment]
healm-chart is ready for use!
